text-recognition/RNMLKitTextRecognition.podspec

```diff
@@ -22,14 +22,14 @@ Pod::Spec.new do |s|
 
   s.dependency "React"
   # To recognize Latin script
-  s.dependency 'GoogleMLKit/TextRecognition', '2.6.0'
+  s.dependency 'GoogleMLKit/TextRecognition', '4.0.0'
   # To recognize Chinese script
-  s.dependency 'GoogleMLKit/TextRecognitionChinese', '2.6.0'
+  s.dependency 'GoogleMLKit/TextRecognitionChinese', '4.0.0'
   # To recognize Devanagari script
-  s.dependency 'GoogleMLKit/TextRecognitionDevanagari', '2.6.0'
+  s.dependency 'GoogleMLKit/TextRecognitionDevanagari', '4.0.0'
   # To recognize Japanese script
-  s.dependency 'GoogleMLKit/TextRecognitionJapanese', '2.6.0'
+  s.dependency 'GoogleMLKit/TextRecognitionJapanese', '4.0.0'
   # To recognize Korean script
-  s.dependency 'GoogleMLKit/TextRecognitionKorean', '2.6.0'
+  s.dependency 'GoogleMLKit/TextRecognitionKorean', '4.0.0'
```

Comment:
Let's extract the dependency updates part into another PR updating dependencies for all the packages as they need to be kept in-sync to not break apps that use multiple packages.